certora/specs/SummaryLib.spec

```
@@ -86,23 +86,23 @@ function mockPrice() returns uint256 {
     return updatedPrice;
 }
 
-function positionAsAssets (address borrower) returns (uint256, uint256) {
-    uint256 mTotalBorrowAssets;
-    uint256 mTotalBorrowShares;
 
+// Ensure this function is only used when no interest is accrued,
+// or enforce that the last update matches the current timestamp.
```

```suggestion
// Ensure this function is only used when no interest is accrued, or enforce that the last update matches the current timestamp.
```
(optional)  because we don't have a formatter yet on CVL files, so we can keep the "one line per sentence" rule

---

Implemented here https://github.com/morpho-org/pre-liquidation/pull/37/commits/8c4110a8134c977d27abc287859c44e362cde28e.